.identity/03_managed_identity_ci.tf

```diff
@@ -0,0 +1,18 @@
+module "identity_ci" {
+  source = "github.com/pagopa/terraform-azurerm-v3//github_federated_identity?ref=v7.34.0"
+
+  prefix    = var.prefix
+  env_short = var.env_short
+  domain    = var.domain
+
+  identity_role = "ci"
+
+  github_federations = var.ci_github_federations
+
+  cd_rbac_roles = {
```

Comment:
`ci_rbac_roles`

Reply:
fixed in https://github.com/pagopa/io-web-profile-backend/pull/32/commits/4f8a7668f8d9035acf20e7ef30df8b6cd26ccc3d